superset/migrations/versions/0d0c0e69e87a_add_data_access_policy_agree_date.py

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Add data_access_policy_agree_date to UserAttribute
+
+Revision ID: 0d0c0e69e87a
+Revises: afc69274c25a
+Create Date: 2019-06-05 14:46:21.059509
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '0d0c0e69e87a'
+down_revision = 'afc69274c25a'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column(
+        'user_attribute',
+        sa.Column('data_access_policy_agree_date', sa.DateTime(), nullable=True))
```

Comment:
It sees like `True` is the default per [here](https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/sql/schema.py#L1186) and thus probably could be removed.